### PR TITLE
determine which day is displayed after login

### PR DESCRIPTION
**Pull request by benhummel**
- fixed a bug where dates weren't getting displayed correctly
- added a check for whether today's date is within dates of known shows

### CODE REVIEW

File: views/userPage.html [hunk 1]
```diff
@@ -77,27 +77,47 @@
         //   vex.dialog.confirm({ message: 'Are you absolutely sure you want to destroy the alien planet?', callback: function(value) { console.log("Yes")}
         // })
         // };
-        vex.dialog.alert({ unsafeMessage: 'Here\'s your SXSW schedule for the week, pulled from your top Spotify artists and their related artists. <br /><br />✌🏽✌🏻,<br />- Will + Mathew'})
+        vex.dialog.alert({ unsafeMessage: 'Here\'s your SXSW schedule for the week, pulled from your top Spotify artists and their related artists. <br /><br />✌🏽✌🏻,<br />- Ben + Mathew'})
```
Comment (benhummel) [inline views/userPage.html]: the most important change

File: views/userPage.html [hunk 2]
```diff
@@ -77,27 +77,47 @@
...
+            if(today.getTime()<firstShowDate.getTime() | today.getTime()>lastShowDate.getTime()) {
+              console.log("it's not SXSW :(");
+            } else {
+            // If today is within SXSW dates, start on appropriate day
+            // TODO: if no shows are on this day, skip to next day
+              console.log("it's SXSW! :D");
+              for(var i=0;i<results.length;i++){
+                var showDate = allShowDates[i];
+                // If the show day is earlier than today, then move to the next day
+                if(today.getDate()-showDate.getDate()>0){
```
Comment (mathewantony31) [inline views/userPage.html]: you could simplify this to `if(today>showDate)`

File: views/userPage.html [hunk 3]
```diff
@@ -77,27 +77,47 @@
...
+            // determine date parameters for SX shows
+            var firstShowDate = new Date(allShowDates[0]);
+            var lastShowDate = new Date(allShowDates[allShowDates.length - 1]);
 
-            // Hide all days
-            for(var i=0;i<days.length;i++){
-              $('#'+days[i]).hide();
+            // If today is not within SXSW dates, just start on day 0
+            // use getTime() to convert date to milliseconds for easy math 
+            if(today.getTime()<firstShowDate.getTime() | today.getTime()>lastShowDate.getTime()) {
```
Comment (mathewantony31) [inline views/userPage.html]: could simplify to `if(today<firstShowDate | today>lastShowDate)`

File: views/userPage.html [hunk 4]
```diff
@@ -77,27 +77,47 @@
...
+              console.log("it's not SXSW :(");
+            } else {
+            // If today is within SXSW dates, start on appropriate day
+            // TODO: if no shows are on this day, skip to next day
```
Comment (mathewantony31) [inline views/userPage.html]: not sure if I'm missing something but, based on how you've set this up, this is actually solved yeah?

File: views/userPage.html (hunk 3 above)
Comment (benhummel) [reply views/userPage.html]: Thanks! Thought I needed to convert to time to compare for some reason. Seems to work just fine without as you said.

File: views/userPage.html (hunk 4 above)
Comment (benhummel) [reply views/userPage.html]: Honestly, I'm not sure. We at least need to test it.

**Pull request merged**